src/Components/Form/Checkbox.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Checkbox = ({ header, id, checked, handler }) => (
+  <label htmlFor={id} className="todo__label">
+    {header && `${header}: `}
+    <input
+      id={id}
+      className="todo__checkbox"
+      type="checkbox"
+      checked={checked}
+      onChange={e => handler(e.target.id, !checked)}
```

Comment:
```suggestion
      onChange={e => statusChangeHandler(e.target.id, !checked)}
```
что хендлит эта функция?

Reply:
У меня три компонента: Input, Checkbox и Select. В каждом есть handler (по сути - это одна и та же функция). Не уверен, что стоит её именовать по разному в разных местах.